test/api-tests.js

```diff
@@ -1,49 +1,245 @@
+                "geneSummary":null, "primaryIdentifier":"S000002859"}
+        ],
+            "wasSuccessful":true,
+            "error":null,
+            "statusCode":200};
+
+
+        server.respondWith([
+            200,
+        {"Content-Type": "application/json" }, JSON.stringify(testObject)]);
+
+
+        global.window.api.getYeastMineInfo(query).then((data) => {
+            expect(data.results[0].description).to.equal(testObject.results[0].description);
+            done();
+        }).catch(() => {
+            done();
+        });
+
     });
-});
 
-describe("JASPAR API", function () {
-    it("should load", function () {
+    it("makes the correct call to YeastMine (general data)", done => {
+
+        const testObject = {"results":[
+            {"symbol":"YHP1", "length":1062,
+                "description":"Homeobox transcriptional repressor",
+                "geneSummary":null, "primaryIdentifier":"S000002859"}
+        ],
+            "wasSuccessful":true,
+            "error":null,
+            "statusCode":200};
+
+
+        server.respondWith([
+            200,
+        {"Content-Type": "application/json" }, JSON.stringify(testObject)]);
+
+
+        global.window.api.getYeastMineInfo(query).then((data) => {
+            expect(data.results[0].description).to.equal(testObject.results[0].description);
+            done();
+        }).catch(() => {
+            done();
+        });
+
     });
-});
 
-describe("SGD API", function () {
-    it("should load", function () {
+    it("makes the correct call to YeastMine (regulation info)", done => {
+
+        const testObject = [
+            {properties:
+                {id: 6393710}},
+            {properties:
+                {id: 6393710}}
+        ];
+
+
+
+        server.respondWith([
+            200,
+        {"Content-Type": "application/json" }, JSON.stringify(testObject)]);
+
+
+        global.window.api.getRegulationInfo(query).then((data) => {
+            expect(data[0].properties.id).to.equal(data[0].properties.id);
+            expect(data.length).to.equal(2);
+            done();
+        }).catch(() => {
+            done();
+        });
+
     });
-});
 
-describe("NCBI Gene API", function () {
-    it("should load", function () {
+    it("makes the correct call to YeastMine (gene ontology info)", done => {
+
+        const testObject = [
+            {properties:
+                {id: 6393710}},
+            {properties:
+                {id: 6393710}}
+        ];
+
+
+
+        server.respondWith([
+            200,
+        {"Content-Type": "application/json" }, JSON.stringify(testObject)]);
+
+
+        global.window.api.getGeneOntologyInfo(query).then((data) => {
+            expect(data[0].properties.id).to.equal(data[0].properties.id);
+            expect(data.length).to.equal(2);
+            done();
+        }).catch(() => {
+            done();
+        });
+
     });
-}); */
+
+});
```

Comment:
Have you done an `npm test` on this branch to ensure that these (and the ones below) are integrated and working?

Reply:
Yes, they all are working